recipes/arrays.md

```diff
@@ -42,7 +42,7 @@ const reduced = items.reduce((acc, item) => {
   return acc;
 }, {});
 
-items.reverse();
+const reversed = items.reverse();
```

Comment:
the intent here was to accentuate that `.reverse()` doesn't return new collection, but rather modifies original in place. On the other had dart example has an assignment, so perhaps it worth keeping this example consistent on both sides